vendor/github.com/grafana/dskit/ring/util.go

```diff
@@ -158,54 +155,3 @@ func searchToken(tokens []uint32, key uint32) int {
 	}
 	return i
 }
-
-// GetFirstAddressOf returns the first IPv4 address of the supplied interface names, omitting any 169.254.x.x automatic private IPs if possible.
-func getFirstAddressOf(names []string, logger log.Logger) (string, error) {
```

Comment:
hmm here you're changing vendor code :thinking:

Reply:
Sorry, I wanted to open it as a draft PR. The corresponding upstream PR in dskit is https://github.com/grafana/dskit/pull/100
It would also need to be used in Cortex.

Reply:
oops, my bad.